maven-java/maven-archetype-akkasls-event-sourced-entity/src/main/resources/archetype-resources/src/main/proto/myentity_api.proto

```diff
@@ -7,26 +7,26 @@ import "google/api/annotations.proto";
 
 package $package;
 
-option java_outer_classname = "MyEntityApi";
+option java_outer_classname = "CounterApi";
 
 message SetValueCommand {
```

Comment:
We should aim for the same commands we added for the value entity counter, eg: 

```proto
message IncreaseValue { // <4>
  string counter_id = 1 [(akkaserverless.field).entity_key = true]; // <5>
  int32 value = 2;
}

message DecreaseValue {
  string counter_id = 1 [(akkaserverless.field).entity_key = true];
  int32 value = 2;
}

message ResetValue {
  string counter_id = 1 [(akkaserverless.field).entity_key = true];
}

message GetCounter {
  string counter_id = 1 [(akkaserverless.field).entity_key = true];
}
```

and corresponding events, `ValueIncreased`, `Reset`, etc.

Reply:
done now